packages/back-end/src/models/InformationSchemaTablesModel.ts

```diff
@@ -0,0 +1,63 @@
+import omit from "lodash/omit";
+import z from "zod";
+import mongoose from "mongoose";
+import { InformationSchemaTablesInterface } from "../types/Integration";
+import { errorStringFromZodResult } from "../util/validation";
+import { logger } from "../util/logger";
+
+const informationSchemaTablesSchema = new mongoose.Schema({
+  organization: String,
+  table_name: String,
+  table_schema: String,
+  database_name: String,
```

Comment:
Do we need any indexes for when we query for tables?

Reply:
Added indexing to the organizations for this model, and the `InformationSchemasModel`.